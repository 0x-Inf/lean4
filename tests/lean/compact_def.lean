Definition f x y := x + y
Definition g x y := sin x + y
Definition h x y := x * sin (x + y)
Show Environment 3