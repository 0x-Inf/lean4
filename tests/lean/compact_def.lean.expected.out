  Set: pp::colors
  Set: pp::unicode
  Imported 'specialfn'
  Defined: f
  Defined: g
  Defined: h
Definition f (x y : ℕ) : ℕ := x + y
Definition g (x y : ℝ) : ℝ := sin x + y
Definition h (x y : ℝ) : ℝ := x * sin (x + y)
